packages/cli/test/integration.test.ts

```diff
@@ -2413,11 +2482,11 @@ test('create zero-config deployment', async () => {
 
   expect(data.error).toBe(undefined);
 
-  const validBuilders = data.builds.every(build =>
+  const validBuilders = data.builds.every((build: Build) =>
     isCanary ? build.use.endsWith('@canary') : !build.use.endsWith('@canary')
   );
 
-  const buildList = JSON.stringify(data.builds.map(b => b.use));
+  const buildList = JSON.stringify(data.builds.map((b: Build) => b.use));
```

Comment:
Might be better to change line 2481 instead to be `JSON.parse(text) as SomeTypeHere`

Reply:
Good point. I looked for other `JSON.parse` uses and made sure the result wasn't `any`. Changes here: [`5acb8bf` (#9672)](https://github.com/vercel/vercel/pull/9672/commits/5acb8bf06d37985ee09d6a001c38f7664e6f1732)